Dockerfile

```diff
@@ -2,20 +2,16 @@ FROM flyimg/docker-app
 
 ENV TZ=Europe/Amsterdam
 
+# Remove the list of packages of apt as it can get large in size
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
As this is alread in the extended image (https://github.com/flyimg/docker-app/blob/master/Dockerfile#L13) and we don't install stuff in this image, it can be removed as it doesn't do anything.

It is only useful when directly applying it to the line where we actually install or update packages, for example in the removed line 16 (https://github.com/SkyDreams/flyimg/pull/4/files#diff-3254677a7917c6c01f55212f86c57fbfL16)